readthedocs/docsitalia/serializers.py

```diff
@@ -3,14 +3,37 @@
 from __future__ import absolute_import
 
 from rest_framework import serializers
+from readthedocs.restapi.serializers import ProjectSerializer
 
-from readthedocs.projects.models import Project
 
-
-class ProjectsByTagSerializer(serializers.HyperlinkedModelSerializer):
+class ProjectsByTagSerializer(ProjectSerializer):
 
     """Projects by Tag DRF Serializer"""
 
-    class Meta:
-        model = Project
-        fields = ('name', 'description', 'url')
+    publisher = serializers.SerializerMethodField()
+    publisher_project = serializers.SerializerMethodField()
+
+    class Meta(ProjectSerializer.Meta):
+        fields = (
```

Comment:
visto che ci siamo in un altro commit mettiamo pure i tags

Reply:
Questa la facciamo nell' altra pr? Tanto è aperta su questa

Reply:
no perchè l'altra pr non serve più se metti i tags qua

Reply:
ok

Reply:
Il projectviewset va creato in questa pr perchè non c'è

Reply:
ce l'ho in pronto in un 'altra pr ma mi servono le modifiche hai fatto su questa

Reply:
allora aspetta che pusho :-)